### PR TITLE
Fixes #17892: Typo in network resiliency documentation

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/17892

### CODE REVIEW

File: src/reference/modules/administration/pages/server.adoc [hunk 1]
```diff
@@ -324,14 +324,14 @@ systemctl restart postgresql
 
 === Between nodes and their policy server
 
-When connections between a policy server and its managed nodes breaks:
+When connections between a policy server and its managed nodes break:
```
Comment (amousset) [inline src/reference/modules/administration/pages/server.adoc:327]: maybe "the connection" instead

File: src/reference/modules/administration/pages/server.adoc [hunk 2]
```diff
@@ -324,14 +324,14 @@ systemctl restart postgresql
...
 
 * All current policies continue to be applied by the nodes at defined execution interval
 * Agent continues to generate local logs
 
 === Between root and relay
 
-When connections between a root server and a relay breaks:
+When connections between a root server and a relay break:
```
Comment (amousset) [inline src/reference/modules/administration/pages/server.adoc:334]: here too

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**